.github/workflows/ci.yml

```diff
@@ -37,7 +37,26 @@ jobs:
       run: cargo check --verbose
     - name: Test
       run: cargo test --verbose
-      
+
+  wasm:
+    name: Wasm CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Dependencies
```

Comment:
Not sure that this section is correct, according to this https://rustwasm.github.io/wasm-bindgen/wasm-bindgen-test/continuous-integration.html all it needs to have is this:

```
- name: Install
        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

There is no need to install libraries because wasm target do not use them.

Reply:
Ok

Reply:
I did it